packages/playback-core/src/index.ts

```diff
@@ -139,12 +139,27 @@ export const getType = (props: Partial<Pick<MuxMediaProps, 'type' | 'src'>>) =>
 };
 
 export const getStreamTypeConfig = (streamType?: ValueOf<StreamTypes>) => {
-  if (streamType === StreamTypes.LL_LIVE) {
-    return {
-      maxFragLookUpTolerance: 0.001,
+  const defaultConfig = {
```

Comment:
seems a bit weird that the default config is part of `getStreamTypeConfig`. I'd expect the section below where we instantiate hlsjs to be more like:
```js
new Hls({
  ...defaultConfig,
  debug,
  startPosition,
  ...streamTypeConfig
});
```

Reply:
cool I'm down.